### PR TITLE
Implement horizontal fly speed

### LINKED ISSUE #178: Fix flying speed to match vanilla
Currently the flying speed (in creative mode) feels a lot slower than in vanilla. From a quick look at the decompiled vanilla source code, it looks like horizontal flying speed should be `PlayerAttributes.flyingSpeed` (multiplied by 2 if sprinting). If you update `calculatePlayerSpeed` in `PlayerAccelerationSystem` accordingly, hopefully it'll work 🤞. If you test the flying speed against vanilla and it still doesn't match, you may need to look at the decompiled vanilla source code to figure out why it's not matching. I'm not sure why the vertical flying velocity is so much slower than Delta Client, that part will require a bit more investigation.

I already have most code snippets relevant to player physics pulled out into separate files so that I can refer back to them easily. Contact me on Discord if you want to tackle this issue and I'd be happy to help get you started.

### PR DESCRIPTION
**Pull request by FloofyPlasma**
# Description

This PR implements the horizontal fly speed to match the vanilla client. A frame-by-frame analysis was performed and it matches minus some minor frame timing discrepancies.

Partially fixes #178 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation comments
- [x] My changes generate no new warnings

### CODE REVIEW

File: Sources/Core/Sources/ECS/Systems/PlayerAccelerationSystem.swift [hunk 1]
```diff
@@ -164,8 +168,13 @@ public struct PlayerAccelerationSystem: System {
       )
       let block = world.getBlock(at: blockPosition)
       let slipperiness = block.material.slipperiness
-
+
```
Comment (stackotter) [inline Sources/Core/Sources/ECS/Systems/PlayerAccelerationSystem.swift:171]: Remove the trailing newlines

**Review by stackotter — CHANGES_REQUESTED** [review]
Looks good 👌 Just one formatting change to make (trailing newlines) but other than that it's ready to merge once the workflows finish

**Comment by FloofyPlasma** [comment]
Removed the trailing newline

**Review by stackotter — APPROVED** [review]
Lgtm 🎉

**Pull request merged**